docs/source/howto/installation.rst

```diff
@@ -558,13 +558,13 @@ In order to restore a backup, you will need to:
 
 Managing multiple users
 =======================
-Setups with multiple users for a single AiiDA instance are currently not supported.
-Instead, each AiiDA user should install AiiDA in a Unix/Windows account on their own computer.
-Under this account it is possible to configure all the credentials necessary to connect to remote computers.
-Using independent accounts will ensure that, for instance, the SSH credentials to connect to supercomputers are not shared with others.
+AiiDA currently does not support multiple users running concurrently on the same AiiDA instance (database & file repository).
+While AiiDA will tag any node with the :py:class:`~aiida.orm.users.User` who created it, there is currently no permission system in place to limit the operations that can be performed by a given user.
 
-Data can be shared between instances using :ref:`AiiDA's export and import functionality <how-to:share:archives>`.
-Sharing (subsets of) the AiiDA graph can be done as often as needed.
+AiiDA assumes that each user has AiiDA installed on the operating system account of their computer.
+Data can be shared between private AiiDA instances through :ref:`AiiDA's export and import functionality <how-to:share:archives>`.
```

Comment:
Again, instead of instances, I would maybe favor "data store", if we explain that in the beginning as the collection of database and file system that are configured for a profile where data is stored.

Reply:
will simply replace `instance` by `profile`, which is indeed better